coordinator/internal/transitengineapi/transitengineapi.go

```diff
@@ -79,6 +79,9 @@ func NewTransitEngineAPI(authority stateAuthority, port int, logger *slog.Logger
 					ClientCAs:  meshCAPool,
 					ClientAuth: tls.RequireAndVerifyClientCert,
 					MinVersion: tls.VersionTLS12,
+					GetCertificate: func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
+						return nil, nil
+					},
```

Comment:
The `GetCertificate` addition in 93a91529f65dd2854470f4cf8fa8df215bed92bb does not make sense.